ccgains/trades.py

```diff
@@ -606,6 +650,11 @@ def append_csv(
             according to the locale setting; or it must be a tzinfo
             subclass (from dateutil.tz or pytz)
 
+        :param fill_missing_fees: bool:
+            Indicate whether fees should be obtained with 'blockcypher' in case
+            the csv-file does not contain them. This will be done based on the
+            transaction-ID (TX). (Default: `False`)
```

Comment:
Please add a warning, something like: "Please note that using this introduces a privacy leak: Since 'blockcypher' will be asked about all your withdrawals, they will be able to exactly tell which wallets belong to your IP."